### DISCUSSION TITLE
大佬，请问爬取网页资源链接的代码主要在哪个文件里？

### DISCUSSION THREAD

**Discussion by wenbolics** [xifangczy/cat-catch#246] [Q&A]
** 详细描述想要添 加/修改/增强 的功能 **
我是小白，想要学习一下爬取资源链接的那部分代码，最好能直接在控制台运行。

**Comment by xifangczy** [comment]
没有主动爬取的代码。。
原理很简单
https://developer.chrome.com/docs/extensions/reference/webRequest/#event-onResponseStarted
看看这个API就会了。

核心都在 [background.js](https://github.com/xifangczy/cat-catch/blob/master/js/background.js) 很简单...
